in-memory-db.go

```diff
@@ -165,6 +185,37 @@ func handle(c *client) {
 			case "help":
 				write(c.conn, help())
 
+			case "list":
```

Comment:
better separate it in its own command
as `slist` for set list
and `glist` for get list